Geek-Collectors-Network/geek-collectors-network
# 176 create item list UI

### Summary
Created a item list with search functionality; this works identically to the friends list.

### Testing

1. Navigate to `frontend` folder; open a terminal at this location
2. Run `npm run start:dev`
3. Open Chrome and navigate to http://localhost:8080/test
5. Rescale the window; there are three views -- three columns, two columns, one column
6. Open dev tools and click on the delete button; it should show a delete message in the console
7. Try search and clearing search; the grid should respond accordingly.
8. Try refreshing the page; you should see a very brief animated loading icon
